component-samples/map-components/samples/cdn/index.html

```diff
@@ -15,6 +15,11 @@
         padding: 0;
       }
     </style>
+    
+    <!-- Load Calcite Components. Note this could be after loading the ArcGIS Maps SDK for JS but before loading the Coding Components  -->
```

Comment:
```suggestion
    <!-- Load Calcite components. Note this could be after loading the ArcGIS Maps SDK for JavaScript but before loading the Map components  -->
```